src/graphql/query/fundedBorrowerProfile.graphql

```diff
@@ -7,10 +7,6 @@ query fundedBorrowerProfile(
 	$imgRetinaSize: String = "w960h720"
 ) {
 	general {
-		customSort: uiExperimentSetting(key: "funded_lyml_sort") {
```

Comment:
not referenced anywhere else

Reply:
I think I removed usage of this last week with the Funded BP cleanup...